packages/uows-client-generator/src/construct-functions/constructFunctions.ts

```diff
@@ -72,12 +72,10 @@ export const makeArgsObjTemplate: string = `   private makeArgsObj(functName: st
           }\n\n`;
 
 //A string specifying the constructor for the UOWSSoapInterface class
```

Comment:
You could replace this comment with TypeDoc
/**
 * Constructor of the template
 * @param wsdl A string specifying the constructor for the UOWSSoapInterface class
 */